package.json

```diff
@@ -26,6 +26,7 @@
     "url": "git+https://github.com/wordbots/wordbots-core.git"
   },
   "dependencies": {
+    "@babel/core": "^7.0.1",
```

Comment:
What changed that made this required?

Reply:
We updated some other packages with Greenkeeper that now require `@babel/core` instead of `babel-core`. I just bundled the change in with this PR.